{{cookiecutter.project_slug}}/README.md

```diff
@@ -81,3 +91,34 @@ exec fastlane match development` from `ios/`.
 Not a CB developer? Create an [Apple developer](https://developer.apple.com)
 and follow the instructions on [codesigning guide](https://codesigning.guide/)
 to setup your certificates.
+
+##Blueprints
+
+###Google Maps API
+
+As soon as the project is created, you need to run:
+
+`npm install react-native-maps --save-exact`
+`yarn add react-native-maps -E`
+
+###Camera
+
+Do:
+
+`npm install react-native-camera --save`
+
+##TroubleShooting
+
+###Module "redux" does not exist in the haste module map.
+
+Run the following:
+
+`watchman watch-del-all`
+`rm -rf node_modules && npm install`
+`rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`
+`rm -rf /tmp/haste-map-react-native-packager-*`
+
+Reset the simulator(s), then run:
+
+`npm install`
+`npm install --save redux`
```

Comment:
if you ran above commands we don't need this since redux is already in package.json